packages/create-cloudflare/src/__tests__/common.test.ts

```diff
@@ -6,10 +6,13 @@ import { version as currentVersion } from "../../package.json";
 import {
 	isAllowedExistingFile,
 	isGitConfigured,
+	quoteShellArgs,
 	validateProjectDirectory,
 } from "../common";
 import { isUpdateAvailable } from "../helpers/cli";
 
+vi.mock("process");
```

Comment:
Rather than mocking out process.platform, why not rely upon the fact that our CI will test on three different platforms?

We can have two different tests that are conditional upon the current platform.

https://vitest.dev/api/#test-skipif (or more likely `runIf()`).